src/Port.js

```diff
@@ -0,0 +1,16 @@
+const Ship = require("./Ship");
```

Comment:
This `Ship` is never used in this file, so this can be deleted. Remember the `ship` you see passed into the functions here is just a placeholder to be used inside the function, you're not actually passing in an instance of the Ship object like the one this is importing